python/hail/methods/qc.py

```diff
@@ -234,22 +234,22 @@ def concordance(left, right):
     Table 1: Concordance statistics per column key
 
         - **s** -- column key.
-        - **nDiscordant** (*Long*) -- Count of discordant calls (see below for
+        - **n_discordant** (:py:data:`.tint64`) -- Count of discordant calls (see below for
           full definition).
-        - **concordance** (*Array[Array[Long]]*) -- Array of concordance per
-          state on left and right, matching the structure of the global summary
-          defined above.
+        - **concordance** (:class:`.tarray` of :class:`.tarray` of :py:data:`.tint64`) --
+          Array of concordance per state on left and right, matching the structure of
+          the global summary defined above.
 
     Table 2: Concordance statistics per row key
 
         - **v** -- row key.
-        - **nDiscordant** (*Long*) -- Count of discordant calls (see below for
+        - **n_discordant** (*Long*) -- Count of discordant calls (see below for
           full definition).
         - **concordance** (*Array[Array[Long]]*) -- Array of concordance per
```

Comment:
Shouldn't these types be changed to match those in Table 1?

Reply:
ah, yes.